pokemongo_bot/cell_workers/evolve_pokemon.py

```diff
@@ -26,6 +29,9 @@ def _validate_config(self):
         if isinstance(self.evolve_all, basestring):
             self.evolve_all = [str(pokemon_name).strip() for pokemon_name in self.evolve_all.split(',')]
 
+        if 'evolve_speed' in self.config:
+            self.logger.warning("evolve_speed is deprecated, please use instead 'min_evolve_speed' and 'max_evolved_speed'.")
```

Comment:
@Gzerox We have the same kinds of code to ensure notify users the configuration file format changes. So it's ok.